Fitness/Controllers/HabitTrackingController.swift

```diff
@@ -167,7 +168,8 @@ class HabitTrackingController: UIViewController {
         // BACK BUTTON
         if type != .cardio {
             backButton = UIButton()
-            backButton!.setImage(UIImage(named: "back-arrow-grey"), for: .normal)
+            backButton!.setImage(UIImage(named: "black-yellow-next-arrow"), for: .normal)
```

Comment:
should use nil coalescing instead of force unwrapping
`backButton?.setImage...`

Reply:
true whoops